CHANGELOG.md

```diff
@@ -0,0 +1,77 @@
+# go-filecoin changelog
+
+## go-filecoin 0.1.4
+
+We're happy to announce go-filecoin 0.1.4. This release contains a better install experience, Proof-of-Spacetime integration, more reliable message sending and networking, and many other improvements. Get pumped! 🎁
```

Comment:
initial Proof of Spacetime integration? it is not the final construction

Reply:
done